src/operations/stats.ts

```diff
@@ -130,7 +136,11 @@ export interface CollStats extends Document {
   scaleFactor: number;
 }
 
-/** @public */
+/**
+ * @public
+ * @deprecated the `collStats` operation will be removed in the next major release.  Please
```

Comment:
I think since this interface isn't obviously related to just the collstats operation, we should call out the fact that it is (only used for the collstats operation which will be removed)